package.json

```diff
@@ -47,11 +49,9 @@
     "rimraf": "^5.0.1",
     "rxjs": "^7.5.6",
     "swagger-ui-express": "^5.0.0",
-    "update": "^0.7.4",
```

Comment:
this package contains critical vulnerabilities and i wasn't able to updgrade in satisfying way. Also i didn't find a case in which it is needed (if you know some pls tell me) so i decided to drop it.